pandas_gbq/constants.py

```diff
@@ -0,0 +1,8 @@
+# Copyright (c) 2024 pandas-gbq Authors All rights reserved.
+# Use of this source code is governed by a BSD-style
+# license that can be found in the LICENSE file.
```

Comment:
just double checking that this is normal given that we use a difference license header for other Google stuff

Reply:
Not normal, but expected. pandas-gbq was originally split off from pandas itself, so we can't use the Google header. It's not 100% Google's copyright since not all contributions were under the CLA.